wallet/res/values-th/strings.xml

```diff
@@ -65,8 +65,8 @@
     <string name="send_coins_fragment_pin_label">ใส่รหัส PIN</string>
     <string name="send_coins_fragment_hint_fee_economic">ค่าธรรมเนียมระบบเครือข่ายของ %s จะถูกจ่าย</string>
     <string name="send_coins_fragment_hint_fee_priority">ค่าธรรมเนียมพิเศษของ %s จะถูกจ่าย หากคุณสนใจค่าธรรมเนียมพิเศษนี้ โปรดใช้ \'priority\' เมื่อต้องการการยืนยันอย่างรวดเร็ว</string>
-    <string name="send_coins_fragment_hint_dusty_send">จำนวนเงินน้อยเกินไปที่จะส่ง</string>
-    <string name="send_coins_fragment_hint_insufficient_money">ยอดเงินไม่เพียงพอ</string>
+    <string name="send_coins_error_dusty_send">จำนวนเงินน้อยเกินไปที่จะส่ง</string>
+    <string name="send_coins_error_insufficient_money">ยอดเงินไม่เพียงพอ</string>
```

Comment:
Luckily the memory function of Transifex will handle this change.

Reply:
Though I see that these strings are actually moved to `common` -- which is also fine.  The translation update after this will also move all other languages for these strings to `common`.